CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - The `WithoutTimestamps` option to `go.opentelemetry.io/otel/exporters/stdout/stdoutmetric` to sets all timestamps to zero. (#3828)
+- Added a log message warning about usage of `SimpleSpanProcessor` in production environments. (#1875)
```

Comment:
This should be the number of the PR fixing the issue, not the related issue.

```suggestion
- Added a log message warning about usage of `SimpleSpanProcessor` in production environments. (#3854)
```

Reply:
Ah, my bad.